src/makeRobotAccountant.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  const errorMessage = 'Bzzz... Error!';
+
+  return (num1) => {
+    counter++;
+
+    const isMoreThen3 = counter > 3;
+    const isOdd = counter % 2 === 0;
+
+    const messageOrNum = isMoreThen3 && isOdd
+      ? message => errorMessage
```

Comment:
In the case of an error, we do not need to have arguments to the function input.
```suggestion
      ? () => errorMessage
```